app/scripts/lib/metrics.js

```diff
@@ -99,43 +95,40 @@ define(function (require, exports, module) {
 
     this._window = options.window || window;
 
-    this._lang = options.lang || 'unknown';
-    this._context = options.context || Constants.CONTENT_SERVER_CONTEXT;
-    this._entrypoint = options.entrypoint || NOT_REPORTED_VALUE;
-    this._migration = options.migration || NOT_REPORTED_VALUE;
-    this._service = options.service || NOT_REPORTED_VALUE;
+    this._able = options.able;
```

Comment:
I found it difficult to navigate this mess, so I sorted most items alphabetically.